index.html

```diff
@@ -3782,7 +3782,7 @@ <h2>Devices in a Local Network Controlled from a Cloud Service</h2>
           access all the <a>Things</a> behind the NAT device,
           assuming the local <a>Intermediary</a>
           has used a converged protocol that can traverse the NAT device
-          and has some way to expose this service to the Internet (STUN, TURN, DyDNS, etc.).
+          and has some way to expose this service to the Internet (STUN, TURN, DynDNS, etc.).
```

Comment:
"DynDNS" is the name of a specific company's Dynamic DNS service.
How about using "DDNS" or "Dynamic DNS"?

Reply:
done - used "Dynamic DNS"